libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -33,26 +36,36 @@
     textToSpeech = new TextToSpeech(context, new TextToSpeech.OnInitListener() {
       @Override
       public void onInit(int status) {
-        if (status != TextToSpeech.ERROR) {
-          textToSpeech.setLanguage(new Locale(language));
+        boolean ableToInitialize = status != TextToSpeech.ERROR && language != null;
+
+        if (ableToInitialize) {
```

Comment:
Why not using a [guard clause](http://wiki.c2.com/?GuardClause) instead?
In the exceptional case (`ableToInitialize` is `false`) log and return and in the normal case call `initializeWithLanguage`. IMO it makes clearer the intention of the method.